doc/cprover-manual/cbmc-tutorial.md

```diff
@@ -348,4 +350,3 @@ comes with a small set of definitions, which includes functions such as
 
 We also have a [list of interesting applications of
 CBMC](http://www.cprover.org/cbmc/applications/).
-
```

Comment:
Shouldn't there be a newline here?

Reply:
This was auto-removed on save, because there are two new lines at the end of the file